.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           at: .
       - run: |
           echo 'export FOLDER_NAME=site-${CIRCLE_BRANCH}' >> $BASH_ENV
-          echo 'export PREVIEW_URL=http://${BUCKET_NAME}.s3-website-eu-west-1.amazonaws.com/${FOLDER_NAME}/${WEBSITE_INDEX_DOCUMENT}' >> $BASH_ENV
+          echo 'export PREVIEW_URL=http://${BUCKET_NAME}.s3-website-eu-west-1.amazonaws.com/${FOLDER_NAME}/en/${WEBSITE_INDEX_DOCUMENT}' >> $BASH_ENV
```

Comment:
This is under the `production ` element, should it not be under `pullrequest`?

Reply:
This PR is dead, too much has changed since.
But I'll take a look when we approach #1016 again, thanks!

Reply:
If so, better cancel it.